src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PathologyRecordMapperTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
+
+import org.hl7.fhir.dstu3.model.Enumerations.AdministrativeGender;
+import org.hl7.fhir.dstu3.model.HumanName;
+import org.hl7.fhir.dstu3.model.Practitioner;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.fhir.PathologyRecord;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static uk.nhs.digital.nhsconnect.lab.results.fixtures.FhirFixtures.generateRequester;
+
+@ExtendWith(MockitoExtension.class)
+class PathologyRecordMapperTest {
+
+    @Mock
+    private PractitionerMapper practitionerMapper;
+
+    @InjectMocks
+    private PathologyRecordMapper pathologyRecordMapper;
+
+    @Test
+    void testMapMessageToPathologyRecord() {
+        final Message message = new Message(new ArrayList<>());
+
+        when(practitionerMapper.mapRequester(message)).thenReturn(
+                Optional.of(generateRequester("Dr Bob Hope", AdministrativeGender.MALE))
+        );
+
+        final PathologyRecord pathologyRecord = pathologyRecordMapper.mapToPathologyRecord(message);
+
+        Practitioner performer = pathologyRecord.getRequester();
+        assertThat(performer.getName())
```

Comment:
Like in `BundleMapperTest.java`, could we use `assertAll` to make sure that all failures are caught
```suggestion
        assertAll(
            () -> assertThat(performer.getName())
                .hasSize(0)
                .first()
                .extracting(HumanName::getText)
                .isEqualTo("Dr Bob Hope"),
            () -> assertThat(performer.getGender().toCode())
                .isEqualTo("female")
        );
```

Reply:
Done 👍